data/json/recipes/other/materials.json

```diff
@@ -32,8 +32,7 @@
       [ [ "forge", 80 ], [ "char_forge", 80 ] ]
     ],
     "components": [
```

Comment:
Linting requires a Perl installation with JSON module, it's not trivial to run on Windows.
Hence why I usually recommend a file that has proper formatting - it's easier than giving a whole tutorial on how to set up Perl.